Snowplow/Internal/Tracker/SPTracker.h

```diff
@@ -246,7 +248,8 @@ NS_SWIFT_NAME(Tracker)
 /*! @brief The subject used to represent the current user and persist user information. */
 @property (readonly, nonatomic, strong) SPSubject * subject;
 /*! @brief The object used for sessionization, i.e. it characterizes user activity. */
-@property (nonatomic, retain) SPSession * session;
+@property (nonatomic, retain) SPSession * session_v1 __deprecated_msg("Use `session` instead. It will be removed in the version 3.0");
```

Comment:
I think it might be worth biting the bullet here and removing the old session as a public object. Renaming is to `session_v1` is a breaking change anyway so might aswell just delete it and then mention it in the blog/discourse posts.